source/RialtoGStreamerMSEAudioSink.cpp

```diff
@@ -377,6 +390,16 @@ static void rialto_mse_audio_sink_get_property(GObject *object, guint propId, GV
         g_value_set_int(value, streamSyncMode);
         break;
     }
+    case PROP_FADE_VOLUME:
+    {
+        if (!client)
+        {
+            g_value_set_uint(value, priv->fadeVolume);
+            return;
+        }
+        g_value_set_uint(value, client->getVolume());
```

Comment:
get Volume returns a double, you need to converse to uint for fade-volume property. Same above if you use one member.